JS_2/task3.js

```diff
@@ -0,0 +1,22 @@
+// Преобразовать Task 2* таким образом, чтобы age принимался через prompt в привязанной вёрстке, а результат выводился в alert
+const age = +prompt("Enter your age, please")
```

Comment:
No checks for invalid value from prompt. Please provide validations according to requirements

Reply:
do you mean type of value, more than zero?